blog/community-update-wasmcloud-incubation-process/index.md

```diff
@@ -0,0 +1,24 @@
+---
+slug: community-update-wasmcloud-incubation-process
+title: 'Community Update: wasmCloud Incubation Process'
+image: '/img/cncf-wasmcloud-in-telecoms/cncf-wasmcloud-in-telecoms.jpg'
```

Comment:
Just used an old image as a placeholder but open to suggestions

Reply:
quite like the new line drawings a la...

<img width="228" alt="wasmCloud Line Logo" src="https://github.com/wasmCloud/wasmcloud.com/assets/113366164/6c72c7dd-515d-4474-a80f-147e4e75103e">